code/game/objects/items/plunger.dm

```diff
@@ -2,8 +2,7 @@
 	name = "plunger"
 	desc = "This is possibly the least sanitary object around."
 	icon = 'icons/obj/items/plunger.dmi'
-	icon_state = "plunger_black"
-	item_state = "plunger_black"
+	icon_state = ICON_STATE_WORLD
```

Comment:
Mask state is not working despite the other states working and the state name being identical to other masks - quite confused.

Reply:
Icon works fine on mask items and mask items break on the plunger. Doesn't seem to be calling the overlay proc.

Reply:
It was hard typechecks in human mask icon update code. :I